src/sagas/base-data.js

```diff
@@ -90,10 +92,13 @@ export function* fetchAllTemplates () {
     // Map template attribute derived config values to the templates
     for (const template of templatesInternal) {
       const customCompatVer = template.customCompatibilityVersion
-
-      template.cpuOptions = customCompatVer
-        ? yield mapCpuOptions(customCompatVer, template.cpu.arch)
-        : null
+      if (customCompatVer) {
+        template.cpuOptions = yield mapCpuOptions(customCompatVer, template.cpu.arch)
+        template.isCopyPreallocatedFileBasedDiskSupported = yield mapConfigKeyVersion('CopyPreallocatedFileBasedDiskSupported', customCompatVer, null)
```

Comment:
Since the value is dependent on compatibility version, `null`  causes the code that uses it to fall back to the setting on the cluster.  It is the same pattern as used by `cpuOptions`.  So this bit of code makes the default to be held in the cluster if the template doesn't define it for any reason:

https://github.com/oVirt/ovirt-web-ui/blob/caa8e71b4abd71ef139188e936ffd871e9b80887/src/sagas/vmChanges.js#L321-L326